### DISCUSSION TITLE
SB Learn Schema

### DISCUSSION THREAD

**Discussion by manjirisulekavadi** [Sunbird-Lern/Community#55] [Documentation]
I am trying to understand the schema for Learn (especially User and Org related tables and relations) It's available at 

https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/schema

Can I have it in a suitable format where I can zoom it as required?

**Comment by manjirisulekavadi** [comment]
@krishnaj20 I didnt get any answer to the schema documentation related question. I am looking for something like Shiksha documentation. If it is not available, we at Tekdi can take it up to contribute to SB documentation.  eg. https://shikshaplatform.io/student

**Reply by Krishnaj20** [reply]
@manjirisulekavadi - @rhwarrier & @reshmi-nair are from Sunbird Lern community. Tagging them so that they can address your query.

**Reply by reshmi-nair** [reply]
@manjirisulekavadi Yes I agree, Schema documentation is not very clear in UserOrg section of Lern Microsite. So as you sugested, if Tekdi can contribute that would be great. Pls find the github link : [UserOrg Schema Page](https://github.com/Sunbird-Lern/Community/blob/main/learn/product-and-developer-guide/user-and-org-service/schema.md)

**Reply by manjirisulekavadi** [reply]
@reshmi-nair delayed response. Thanks for the link and yes will be happy to contribute back. But of course a SPoC for inputs, queries and reviews will help to take this further.
